src/main/java/com/battleship/backend/controllers/GameController.java

```diff
@@ -49,21 +49,27 @@ Boardable placeShip(@RequestBody Request placeRequest) throws Exception{
         }
     }
 
+    // change to return message instead of board.
```

Comment:
I don't think this comment is necessary to check into source control. This is something you can point out in the MR but it's not something that needs to live in the code.

Reply:
Removed: Will put a note in the PR description.